salvo/src/lib/api/env.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package salvo;
+
+// Capture all Environment variables required for the benchmark
+message EnvironmentVars {
+  // Specify the IP version for tests
+  oneof envoy_ip_test_versions {
+    bool v4only = 1;
+    bool v6only = 2;
+    bool all = 3;
+  }
+
+  // Controls whether envoy is placed between the nighthawk client and server
+  string envoy_path = 4;
+
+  // Specify the output directory for nighthawk artifacts
+  string output_dir = 5;
+
+  // Specify the directory where external tests are located
+  string test_dir = 6;
+
+  // Additional environment variables that may be needed for operation
+  map<string, string> variables = 7;
```

Comment:
Same comment about proto maps here, unless we are optimizing, let's prefer repeated fields.